.github/workflows/build_with_conda.yml

```
@@ -0,0 +1,39 @@
+name: Build with conda
+on:
+  push:
+  pull_request:
```

[`pull_request` as a webhook event](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request) not only notifies for the same activity types as what `push:` does, but it does also notify of other potentially unrelated activity types (like reviews request or comment on the PR).

To me, this is a bit too broad, and `push:` should suffice.

```suggestion
  push:
```

What do you think?